apps/ios/Shared/Views/Chat/ChatItem/FramedItemView.swift

```diff
@@ -26,79 +26,51 @@ struct FramedItemView: View {
     @State var imgWidth: CGFloat? = nil
     @State var metaColor = Color.secondary
     @State var showFullScreenImage = false
-
+    @Binding var revealed: Bool
+    
     var body: some View {
+        if chatItem.meta.itemDeleted && !revealed {
+            MarkedDeletedItemView(chatItem: chatItem, showMember: showMember)
```

Comment:
if this is how it worked out, it's better to move it one level up, where you decide if you need FramedItemView